{{cookiecutter.project_slug}}/Readme.md

```diff
@@ -12,7 +12,7 @@ This will generate the requirements/*.txt files, and you should check these in t
 
 If you need to update version pins for your project:
 
-* make or add the change in the appropriate requrements/*.in file
+* make or add the change in the appropriate requirements/*.in file
```

Comment:
```suggestion
* make changes in the appropriate requirements/*.in file
```

Reply:
How should a developer know what the appropriate requirements files are? We should give some guidance on when to update `base.in` and `local.in`. Dev dependencies should not be added to `base.in`, for instance.